.typos.toml

```diff
@@ -256,3 +256,7 @@ extend-ignore-re = [
   "\\WGAM\\W",
   "gam_[a-z]",
 ]
+
+[type.dart.extend-words]
```

Comment:
I don't think we want to copy all the allowlist items from `[type.rust.extend-words]` above. We likely just want to remove the `rust` qualifier? And perhaps add language sections for very specific items.

Reply:
Agree